### ISSUE TITLE
Modal UI causes semantic and accessibility issues

### ISSUE THREAD

**Issue by edoparearyee** [Web3Auth/Web3Auth#102]
We are currently integrating web3auth into our website. However we have come across an issue where due to the modal content being injected onto the page this is causing some semantic issues for our html which could effect SEO rankings.

For example the [`Header`](https://github.com/Web3Auth/Web3Auth/blob/master/packages/ui/src/components/Header.tsx) component includes a `<h1>` element and as our content already has an `<h1>` element this fails our html validation and accessibility checks when checking the rendered markup for the entire page. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#multiple_h1_elements_on_one_page

Initially I thought a solution would be to simply change the element to a `<h2>`. However for a more robust solution would it be possible to wrap the modal content in a `<section>` element as suggested in the link above as the modal is always likely to be secondary to the main content on the page?
